workflows/transcriptomics/rnaseq-pe/rnaseq-pe-tests.yml

```diff
@@ -82,33 +84,25 @@
           asserts:
             has_text:
               text: "FBgn0010247\t13"
-    transcripts_expression:
-      element_tests:
-        GSM461177:
-          asserts:
-            has_text:
-              text: "FBtr0078104\t-\t-\tFBgn0031217\tCG11377\t-\tchr2L:102379-104142\t1583\t1.95689	28.9556	19.9177	37.9936\tOK"
-    genes_expression:
-      element_tests:
-        GSM461177:
-          asserts:
-            has_text:
-              text: "FBgn0031217\t-\t-\tFBgn0031217\tCG11377\t-\tchr2L:102379-104142\t-\t-\t28.9556	19.7218	38.1895\tOK"
     both strands coverage:
       element_tests:
         GSM461177:
             has_size:
               value: 9885639
               delta: 900000
-    negative strand coverage:
+    stranded coverage:
       element_tests:
-        GSM461177:
+        GSM461177_reverse:
             has_size:
               value: 7756965
               delta: 700000
-    positive strand coverage:
-      element_tests:
-        GSM461177:
+        GSM461177_forward:
             has_size:
               value: 7756965
               delta: 700000
+    genes_expression_stringtie:
+      element_tests:
+        GSM461177:
+          asserts:
+            has_text:
+              text: "FBgn0031217 	CG11377 	chr2L 	+ 	102380 	104142 	1.003790 	31.826454 	55.206604"
```

Comment:
I think we may have to use a regex to account for subtle differences here, but I guess we'll see in the test results.

Reply:
At least in SR it was highly reproducible... Let see.